charts/rekor/Chart.yaml

```diff
@@ -1,10 +1,12 @@
 apiVersion: v2
 name: rekor
-description: Part of the sigstore project, Rekor is a timestamping server and transparency log for storing signatures, as well as an API based server for validation
+description: |
+  Part of the Sigstore project, Rekor is a timestamping server and transparency
+  log for storing signatures, as well as an API based server for validation.
 
 type: application
 
-version: 1.0.0-rc.1
+version: 1.0.0-rc.2
```

Comment:
@cpanato is this the recommended way to update this version, since we might still have to relase an rc.2 in rekor later?

Reply:
I have enabled edits by maintainers on this PR. Also wan't sure on this. But I think there will be anyhow a point where the version and appVersion will live their own lives at some point.

Anyhow, feel free to push as needed on the branch, to get this PR finished.

Don't forget to run the `helm-docs charts/rekort` to update the README.md as well.

Reply:
maybe we can wait till we have the rekor 1.0.0 out

Reply:
Shouldn't we have the Chart.lock in there prior to a 1.0.0 release?

Reply:
agree, but I would drop the other images updates for now, or do it in a follow up PR, want to reduce any possible issues when we rollout at this stage

Reply:
Ok I can make that adjustment.  Should be matter of dropping the commit if I used my normal hygiene when committing 😊

Reply:
@cpanato done. Also fixed some errors as pointed out in the files. Those where introduced in other PRs being merged.